src/localization/locales/en-US.json

```diff
@@ -337,14 +342,85 @@
+        "form_step_data_file_step_description_help_url": "https://terraso.org/help/prepare-your-data-file-for-mapping/",
+        "form_step_set_dataset_step_title": "Select Data from “{{selectedFile.name}}”",
+        "form_step_set_dataset_step_description": "Columns",
+        "form_step_set_dataset_step_lat_lng_descriptionn": "Select the latitude and longitude columns from your dataset.",
+        "form_step_visualize_step_title": "Customize Appearance",
+        "form_step_visualize_step_description": "Choose how your data will appear on the map.",
+        "form_step_annotate_step_title": "Customize Annotations",
+        "form_step_preview_step_title": "Preview",
+        "form_step_preview_step_description": "Zoom and pan the map to set how {{entityType}} members will see it.",
+        "form_step_preview_step_map_description": "After you publish the map, {{entityType}} members can view the map or download it as an image or data file.",
+        "form_step_preview_step_save": "Publish",
+        "added_visualization_config": "The map for {{name}} file has been published.",
+        "visualization_source_data": "<0>Created by {{user}} on {{date}} | SOURCE DATA: <1>{{file.name}}.{{file.resourceType}}</1></0>",
+        "visualization_delete": "Delete",
+        "visualization_delete_confirm_title": "Delete map “{{visualization.title}}“?",
+        "visualization_delete_confirm_message": "Are you sure you wish to delete the map “{{visualization.title}}“",
+        "visualization_confirm_delete": "Delete map",
+        "add_visualization_button": "Make Map",
+        "form_step_annotate_fields.mapTitle": "Map Title",
+        "visualization_share_description": "Click “Share” to send a link to the map. Only members of {{ownerName}} can access it.",
+        "visualization_download_png": "Download PNG",
+        "form_step_visualize_shape_circle": "Circle",
+        "form_step_visualize_shape_square": "Square",
+        "form_step_visualize_shape_hexagon": "Hexagon",
+        "form_step_visualize_shape_triangle": "Triangle",
+        "form_visualization_preview_label": "Preview",
+        "deleted_visualization_config": "{{mapTitle}} has been deleted.",
+        "form_step_annotate_data_points_input_label": "Data point {{index}} label",
+        "visualization_file_load_error": "The file “{{name}}” could not be loaded. Please try again or select another file.",
+        "form_step_set_dataset.form.validation_array_min": "Please select at least one data column to display.",
+        "map_create_document_title": "Create Map: {{name}}",
+        "map_view_document_title": "View Map: {{name}}",
+        "fetchVisualizationConfig.not_found": "Map not found",
+        "addVisualizationConfig.unique": "[TODO] The title “{{body.variables.input.title}}“ is being used by other map, change it and try again."
```

Comment:
@paulschreiber Can you review this string? This is the message the user will see when the title is already being used by other map